pom.xml

```diff
@@ -25,6 +25,7 @@
     </scm>
 
     <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
could you please move this change to a separate commit?

Reply:
done